content/docs/help/contact_us.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "Contact Us"
```

Comment:
Would also call this file/page `Help` - then we can include a section about the link to the docs here too